setup.sh

```diff
@@ -193,14 +193,15 @@ function usage()
   say 'usage: ./setup.sh option'
   say ''
   say 'Where option may be one of the following:'
-  say '--help      | -h     Display this usage message'
-  say "--install   | -i     Install $app_name"
-  say "--uninstall | -u     Uninstall $app_name"
-  say '--skip-checks        Skip checks (use this when packaging)'
-  say '--verbose            Explain what is being done'
-  say '--force              Never prompt'
-  say "--completely-remove  Remove $app_name and all files under its responsibility"
-  say "--docs               Build $app_name's documentation as HTML pages into ./build"
+  say '--help                    | -h     Display this usage message'
+  say "--install                 | -i     Install $app_name"
+  say "--uninstall               | -u     Uninstall $app_name"
```

Comment:
These changes and others in this help should be in its own commit.

Reply:
I didn't understand very well, you say create a new commit just for this change?

Reply:
These changes are not part of this commit, and they have a different context. For this reason, they should have their commit. The only expectation is the `--setup-env-notification`.